gulpfile.js

```diff
@@ -186,8 +186,8 @@ jsFiles.forEach((jsFile) => {
   gulp.task(`build:js:${jsFile}`, bundleTask({ watch: false, label: jsFile, filename: `${jsFile}.js` }))
 })
 
-gulp.task('dev:js', gulp.parallel(...jsDevStrings))
-gulp.task('build:js',  gulp.parallel(...jsBuildStrings))
+gulp.task('dev:js', gulp.series(jsDevStrings.shift(), gulp.parallel(...jsDevStrings)))
```

Comment:
not clear whats going on here, lets make it more readable

Reply:
Will change. Spoiler: `.shift()` returns the first element, and mutates the array to remove the first element. This way, we ensure the first element (inpage.js) is built before all the other scripts, which can be built in parallel, because we need to include it with `brfs` into the `contentscript`, needs to be built at that time.